convert2rhel/checks.py

```diff
@@ -592,11 +592,16 @@ def check_package_updates():
             else "on repositories defined in the %s folder" % reposdir
         )
         logger.warning(
-            "The system has %s packages not updated based %s.\n"
+            "The system has %s package%s not updated based %s.\n"
             "List of packages to update: %s.\n\n"
             "Not updating the packages may cause the conversion to fail.\n"
             "Consider stopping the conversion and update the packages before re-running convert2rhel."
-            % (len(packages_to_update), repos_message, " ".join(packages_to_update))
+            % (
+                len(packages_to_update),
+                "" if len(packages_to_update) == 1 else "s",
+                repos_message,
+                " ".join(packages_to_update),
+            )
```

Comment:
I'd personally just go with the catch-all simple way of saying `package(s)` instead of introducing complexity for the sake of slight readability increase

Reply:
That sounds good. Let me change it.